src/Private/Responses.ps1

```diff
@@ -133,7 +133,7 @@ This is an internal function and may change in future releases of Pode.
 function Write-PodeFileResponseInternal {
     [CmdletBinding()]
     param (
-        [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [Parameter(Mandatory = $true, Position = 0, ValueFromPipeline = $true)]
```

Comment:
This would be another example of one where we could simply remove the ValueFromPipeline attribute instead; it's never used, and means we don't need to alter the function's logic brlow.

Reply:
I'm fine with that. But we are breaking from the previous versions